app/models/reason.rb

```diff
@@ -11,13 +11,13 @@ def tp_percentage
 
     count = posts.where(is_tp: true, is_fp: false).count
 
-    (count.to_f / fast_post_count.to_f).to_f
+    count.to_f / fast_post_count.to_f
```

Comment:
Here and elsewhere, you only need `to_f` on one side -- the other side will automatically be coerced to be a float. So, here you could put `count.to_f / fast_post_count` and get the same result.